.travis.yml

```diff
@@ -49,7 +47,10 @@ notifications:
 jobs:
   include:
     - stage: release
-      script: sbt ++$TRAVIS_SCALA_VERSION! ci-release
+      script:
+      - |
```

Comment:
I may have messed up the multiline YAML syntax here, I'm guessing from some examples I found online

Reply:
🤞